Assets/Scripts/Models/Area/World.cs

```diff
@@ -51,6 +51,11 @@ public World(int width, int height, int depth)
         // Creates an empty world.
         SetupWorld(width, height, depth);
         int seed = UnityEngine.Random.Range(0, int.MaxValue);
```

Comment:
I would call it necessary as it supports development, allowing us to start in the _World scene